README_BN.md

```diff
@@ -6,21 +6,21 @@
 
 ইতিহাস আমাদের দেখায় কীভাবে প্রযুক্তি ব্যক্তিগত স্বাধীনতাকে বৃদ্ধি করে, সৃজনশীলতা এবং নতুনত্ব প্রকাশ করে, যা উন্নতি এবং সাফল্যের দিকে পরিচালিত করে।
 
-প্রারম্ভিক সভ্যতাগুলি লেনদেন ট্র্যাক করার জন্য হিসাবরক্ষণ এবং খাতা ব্যবহার করত, যা ব্যক্তি ও সম্প্রদায়কে তাদের অর্থ ব্যবস্থাপনা এবং সহযোগিতায় সহায়তা করত। প্রিন্টিং প্রেস তথ্যকে সাশ্রয়ী করে, সমালোচনামূলক চিন্তাভাবনা এবং বৈজ্ঞানিক অগ্রগতির মাধ্যমে জ্ঞানকে গণতান্ত্রিক করে  তুললো।

```

Comment:
I think the previous one was better here. it's more readable. Talking about the word 'track'

Reply:
Thanks for your feedback, I've updated it accordingly. If anything else please me know.